snare/cloner.py

```diff
@@ -111,7 +151,16 @@ async def process_link(self, url, level, check_host=False):
             self.logger.error("ValueError while processing the %s link", url)
         return res
 
-    async def replace_links(self, data, level):
+    async def replace_links(self, data: Union[bytes, str], level: int) -> BeautifulSoup:
+        """Replace website links to make them suitable for serving
```

Comment:
what does it mean "suitable"? try to be more concrete, maybe even with example

Reply:
Have clarified it in 59dd4d2. Will add detailed descriptions to this and other functions in the next series of commits.